client/src/pages/auth/Auth.jsx

```diff
@@ -0,0 +1,199 @@
+                      First Name
+                    </Input>
+                    <Input
+                      type="text"
+                      name="LastName"
+                      classStyle="flex-auto"
+                      inputRef={dataRef.lastName}
+                    >
+                      Last Name
+                    </Input>
+                  </div>
+                )}
+                <Input
+                  type="email"
+                  name="email"
+                  classStyle={"md:mb-8"}
+                  inputRef={dataRef.email}
+                >
+                  email
+                </Input>
+                <Input
+                  type="password"
+                  name="password"
+                  classStyle={"md:mb-8"}
+                  inputRef={dataRef.password}
+                >
+                  password
+                </Input>
+                {isSignUp && (
+                  <Input
+                    type="password"
+                    name="confirmPassword"
+                    inputRef={dataRef.confirmPassword}
+                  >
+                    Re-type password
+                  </Input>
+                )}
+                <br />
+                <ButtonPrimary type={"submit"}>
+                  {isSignUp ? "Sign up" : "Login"}
+                </ButtonPrimary>
+              </form>
+              <br />
+              <p className="font-medium text-ivory-200">
+                {isSignUp
+                  ? "Already have an account"
+                  : "Don't have an account yet?"}{" "}
+                <span
+                  className="cursor-pointer hover:text-grotto-100"
+                  onClick={() => setIsSignUp((p) => !p)}
+                >
+                  Click here!
+                </span>{" "}
+              </p>
+              <div className={`${!isSignUp && "md:h-14"}`}></div>
+              <br />
+              <div className="flex justify-center">
+                <p className="font-medium text-ivory-200 w-1/4 rounded-full bg-ivory-100 z-10 ">
+                  OR
+                </p>
+              </div>
+              <div className="relative">
+                <hr className="border-ivory-300 surlignage" />
+              </div>
+              <div className={`${!isSignUp && "md:h-12"}`}></div>
+              <br />
+              <div className="flex justify-center">
+                <div className="flex flex-row justify-between w-3/4">
+                  <BsGoogle
+                    color="grotto-100"
+                    size={50}
+                    className="p-2 fill-grotto-100 border border-ivory-200 rounded 
+              drop-shadow-lg hover:fill-white hover:bg-grotto-100"
+                    onClick={googleAuth}
+                  />
+                  <TfiFacebook
+                    color="grotto-100"
+                    size={50}
+                    className="p-2 fill-grotto-100 border border-ivory-200 rounded 
```

Comment:
maybe border-ivory-300 works better with the linked auth's border's as it's the same colour as the hr's above?